keps/sig-api-machinery/1281-network-proxy/README.md

```diff
@@ -103,7 +103,7 @@ For scalability we will be looking at the number of required open connections.
 Increasing usage of webhooks means we need better than 1 request per connection (multiplexing).
 We also need the tunnel to be tolerant of errors in the requests it is transporting.
 HTTP-Connect only supports HTTP requests and not things like DNS requests.
-We assume that for HTTP URL request,s it will be the proxy which does the DNS lookup.
+We assume that for HTTP URL requests it will be the proxy which does the DNS lookup.
```

Comment:
Since there can be a single HTTP URL request or multiple we should update this to "request(s)".